app/org/sagebionetworks/bridge/services/ParticipantService.java

```diff
@@ -154,6 +155,7 @@ public StudyParticipant getParticipant(Study study, Account account, boolean inc
         builder.withExternalId(lookup.getString(EXTERNAL_IDENTIFIER));
         builder.withDataGroups(lookup.getStringSet(DATA_GROUPS));
         builder.withLanguages(lookup.getOrderedStringSet(LANGUAGES));
+        builder.withTimeZone(lookup.getTimeZone(TIME_ZONE));
```

Comment:
Yes thanks for spotting this, in merging a conflict yesterday with the refactor of ParticipantService, I missed that. I've added it back.